sql/src/test/java/org/apache/druid/sql/calcite/schema/SystemSchemaTest.java

```diff
@@ -330,10 +330,10 @@ public void testGetTableMap()
   public void testSegmentsTable() throws Exception
   {
     // total segments = 6
-    // segments 1,2,3 are published and available
-    // segments 4,5,6  are published but unavailable
-    // segment 3 is published but not served
-    // segment 2 is served by 2 servers, so num_replicas=2
+    // segments 1,2,3 are published but unavailable
+    // segments 4,5,6  are published and available
+    // segment 6 is published but not served
```

Comment:
I don't think this comment about "segment 6" is accurate, the "test1"/"test2"/"test3" segments are all available and used

Reply:
removed this comment